src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = calculate(fromFileName, SUPPLY);
+        int totalBuy = calculate(fromFileName, BUY);
+        int result = totalSupply - totalBuy;
+        writeAccount(toFileName, totalSupply, totalBuy, result);
```

Comment:
The method 'getStatistic' seems to be doing more than one operation: it calculates statistics and also writes to the file. Consider splitting it into two separate methods to adhere to the single responsibility principle.